net-p2p/retroshare/retroshare-0.6.9999.ebuild

```diff
@@ -58,16 +66,26 @@ src_configure() {
 	qConfigs+=( $(usex control-socket '' 'no_')libresapilocalserver )
 	qConfigs+=( $(usex gnome-keyring '' 'no_')rs_autologin )
 	qConfigs+=( $(usex gui '' 'no_')retroshare_gui )
-	qConfigs+=( $(usex settings-api '' 'no_')libresapi_settings )
+	qConfigs+=( $(usex jsonapi '' 'no_')rs_jsonapi )
+	qConfigs+=( $(usex service '' 'no_')retroshare_service )
 	qConfigs+=( $(usex sqlcipher '' 'no_')sqlcipher )
 	qConfigs+=( $(usex webui '' 'no_')libresapihttpserver )
+	qConfigs+=( $(usex xapian '' 'no_')rs_deep_search )
 
 	eqmake5 CONFIG+="${qConfigs[*]}"
 }
 
+src_compile() {
+	use jsonapi && ( nonfatal emake ||
+		elog "Because of JSON API, failure at first emake attempt is non-fatal" )
```

Comment:
How does this make sense? Also, `( ... )` is subshell syntax which is not necessary here.

Reply:
There is a problem in the internal dependency handling in the RetroShare-v0.6.5, so first time one run make it fails, but second time as the dependencies are already solved it works fine. Now it doesn't use the sub-shell syntax.